go/tasks/pluginmachinery/k8s/plugin.go

```diff
@@ -31,6 +31,10 @@ type PluginEntry struct {
 	// Boolean that indicates if this plugin can be used as the default for unknown task types. There can only be
 	// one default in the system
 	IsDefault bool
+	// A list of all task types for which this plugin should be default handler when multiple registered plugins
+	// support the same task type. This must be a subset of RegisteredTaskTypes and at most one default per task type
+	// is supported.
+	DefaultForTaskTypes []pluginsCore.TaskType
```

Comment:
Should the choice of which plugin is the default of which type be defined as a config at the registry level?

Reply:
@EngHabu do you mean in RegisteredTaskTypes above?

Reply:
ping @EngHabu